File: Swoole/examples/chat/src/config/socket.php
<?php
/*
 * @Description: 聊天室配置
 * @Author: franktrue 807615827@qq.com
 * @Date: 2021-07-06 08:53:46
 * @LastEditTime: 2021-07-06 09:31:03
 */

 return [
     'host' => "0.0.0.0",
     'port' => 9501,

     'database' => 'redis',

     'redis' => [
         'scheme' => 'tcp',
         'host' => '0.0.0.0',
         'port' => 6379,
     ],

     'avatar' => [
        './images/avatar/1.jpg',
        './images/avatar/2.jpg',
        './images/avatar/3.jpg',
        './images/avatar/4.jpg',
        './images/avatar/5.jpg',
        './images/avatar/6.jpg',
     ],

     'name' => [
        '科比',
        '库里',
        'KD',
        'KG',
        '乔丹',
        '邓肯',
        '格林',
        '汤普森',
        '伊戈达拉',
        '麦迪',
        '艾弗森',
        '卡哇伊',
        '保罗',
     ]
 ];

File: Go/examples/study/network/socket.go
/*
 * @Description: net包从socket中打开，写入，读取数据
 * @Author: franktrue 807615827@qq.com
 * @Date: 2021-07-23 09:10:29
 * @LastEditTime: 2021-07-23 09:18:35
 */
package main

import (
	"fmt"
	"io"
	"net"
)

func main() {
	var (
		host 		= "www.baidu.com"
		port		= "80"
		remote		= host + ":" + port
		msg	string	= "GET / \n"
		data		= make([]uint8, 4096)
		read 		= true
		count		= 0 
	)
	// 创建一个socket链接
	con, err := net.Dial("tcp", remote)
	// 发送我们的消息，一个http GET请求
	io.WriteString(con, msg)
	// con.Write([]byte(msg))

	// 读取服务器响应
	for read {
		count, err = con.Read(data)
		read = (err == nil)
		fmt.Println(string(data[:count]))
	}
	con.Close()
}